src/client/play.js

```diff
@@ -5,15 +5,37 @@ const concat = require('../transforms/binaryStream').concat
 module.exports = function (client, options) {
   client.once('success', onLogin)
 
+  const mcData = require('minecraft-data')(client.version)
+
   function onLogin (packet) {
     client.state = states.PLAY
     client.uuid = packet.uuid
     client.username = packet.username
-    client.signMessage = (message, timestamp, salt = 0) => {
+    client.signMessage = (message, timestamp, salt = 0n, isCommand, preview) => {
```

Comment:
put new fields at the end to not be breaking to current API

Reply:
Is that not what I did?

Reply:
Ah, this is different from what you put in the types

Reply:
I see. My bad.